rviz_tactile_plugins/src/tactile_visual_base.cpp

```diff
@@ -106,8 +109,13 @@ float TactileVisualBase::mapValue(const ::tactile::TactileValue &value)
   // normalize to range 0..1
   if (mode_ == ::tactile::TactileValue::rawCurrent ||
       mode_ == ::tactile::TactileValue::rawMean) {
-    v = (v - raw_range_.min()) / raw_range_.range();
+    if (raw_range_.range() != 0.0 ) { // if fixed range, and manually set to same value, range could be 0
```

Comment:
A range of zero is handled gracefully by the display (turning taxels to pink).
Why do you think this check is needed here in addition?

Reply:
I remember I was wondering why this was not crashing, but could not find why, so kept it. can obviously be removed if useless here.